baybe/objectives/deprecation.py

```diff
@@ -5,7 +5,7 @@
 import warnings
 from typing import TYPE_CHECKING
 
-from cattrs import override
+import cattrs
```

Comment:
because we cannot import `override` under the same name from both `cattrs` and `typing_extensions`. Since `typing` is a built-in module (and `typing_extensions` is only a temporary replacement for it), I gave precedence for the shorter import from that one and use the longer path-based import one for the third-party module

Reply:
Ok, perfect. Anything else? If not --> approve 🙃